pdf-toolbox/src/test/java/com/lowagie/examples/html/ParseNestedHtmlList.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * $Id: HelloHtml.java 3373 2008-05-12 16:21:24Z xlv $
+ *
+ * This code is part of the 'OpenPDF Tutorial'.
+ * You can find the complete tutorial at the following address:
+ * https://github.com/LibrePDF/OpenPDF/wiki/Tutorial
+ *
+ * This code is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ *
+ */
+
+package com.lowagie.examples.html;
+
+import com.lowagie.text.DocumentException;
+import com.lowagie.text.Element;
+import com.lowagie.text.html.simpleparser.HTMLWorker;
+import com.lowagie.text.html.simpleparser.StyleSheet;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Generates a simple 'Hello World' HTML page.
+ *
+ * @author blowagie
+ */
+
+public class ParseNestedHtmlList {
+
+    /**
+     * Generates an HTML page with the text 'Hello World'
+     *
+     * @param args no arguments needed here
+     */
+    public static void main(String[] args) {
+        System.out.println("Parse Nested HTML List");
+        try {
+            final String htmlText =
```

Comment:
What's the difference between test above and this one?

Reply:
This one uses `HTMLWorker#parseToList()` with a StringReader and produce a list on items.
The previous one uses `HtmlParser#parse()` with a Document to produce a PDF.
 
The first one produced a good PDF document with the nested HTML List. But the problem of this issue is really about the `HTMLWorker#parseToList()` not producing the nested List as expected.

Event if the first test, producing the PDF, is not showing the bug, I choose to keep it as a Non-Regression Test.

Reply:
But why second example contains HTML markup as a String and not as a file?